src/Raven.Server/Documents/Subscriptions/SubscriptionStorage.cs

```diff
@@ -675,6 +675,38 @@ private SubscriptionGeneralDataAndStats GetSubscriptionInternal(SubscriptionStat
             return subscriptionData;
         }
 
+        public int WaitForClusterStabilizationInMs;
```

Comment:
TimeSpan? Then you will not need to convert it few times later on

Reply:
done